fem/linearform_ext.cpp

```diff
@@ -25,7 +25,7 @@ void LinearFormExtension::Assemble()
                "match the number of vector dofs!");
 
    const Array<Array<int>*> &domain_integs_marker = *lf->GetDLFI_Marker();
-   const int mesh_attributes_size = fes.GetMesh()->attributes.Size();
+   const int mesh_attributes_max = fes.GetMesh()->attributes.Max();
```

Comment:
This will fail if `attributes` is empty. In that case, we should set `mesh_attributes_max` to zero (this is pretty common in the examples for BCs, maybe we should factor this logic out to a function).

Reply:
See [5611e7b](https://github.com/mfem/mfem/pull/3240/commits/5611e7b60594ba75400583b85ebdf3b2e90c7ee0) (same approach as `LinearForm::Assemble`).